sdk/storage/Azure.Storage.Common/api/Azure.Storage.Common.netstandard2.0.cs

```diff
@@ -17,6 +17,8 @@ public partial class StorageSharedKeyCredential
         public StorageSharedKeyCredential(string accountName, string accountKey) { }
         public string AccountName { get { throw null; } }
         protected static string ComputeSasSignature(Azure.Storage.StorageSharedKeyCredential credential, string message) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public static Azure.Storage.StorageSharedKeyCredential ParseConnectionString(string connectionString) { throw null; }
```

Comment:
We should not add this.  I think there's a better way to get the credentials today.  We need to come up with a plan for exposing connection string parsing across libraries and it's going to look different.